test/secret.test.ts

```diff
@@ -65,3 +65,100 @@ Array [
 ]
 `);
 });
+
+test('Can create an ssh authentication secret', () => {
```

Comment:
I think the name here is wrong...

```suggestion
test('Can create a service account token secret', () => {
```